cloudinit/config/cc_disk_setup.py

```diff
@@ -133,8 +133,11 @@ def handle(_name, cfg, cloud, log, _args):
             if not isinstance(definition, dict):
                 log.warning("Invalid disk definition for %s" % disk)
                 continue
-
             try:
+                if definition.get("timeout"):
```

Comment:
The docstring at the top of this file is used to generate the documentation for this module; it should also be updated.

Reply:
No problem, thanks